yrs/src/block_store.rs

```diff
@@ -219,24 +218,14 @@ impl ClientBlockList {
         }
     }
 
-    pub fn get(&self, index: usize) -> &Block {
-        let ptr = self.list[index].get();
-        unsafe { &*ptr }
+    pub fn try_get(&self, index: usize) -> Option<BlockPtr> {
+        let block_ref = self.list.get(index)?;
+        Some(BlockPtr::from(block_ref))
     }
 
-    pub fn get_mut(&self, index: usize) -> &mut Block {
```

Comment:
Since we're operating on `BlockPtr` which is a value type, we don't need all of the variants of `get`/`get_mut` (we can mark `BlockPtr` as `mut` at whim).